BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -1,13 +1,125 @@
 /** @file
 
-  Copyright (c) 2016 - 2017, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2016 - 2019, Intel Corporation. All rights reserved.<BR>
   SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
 #include "Stage1A.h"
 
-CONST CHAR8*  mBootloaderName = "Intel Slim Bootloader";
+CONST CHAR8*     mBootloaderName = "Intel Slim Bootloader";
+
+CONST CDATA_BLOB mCfgBlobTmpl = {
+  CFG_DATA_SIGNATURE,
+  sizeof (CDATA_BLOB),
+  0,
+  0,
+  sizeof (CDATA_BLOB),
+  FixedPcdGet32 (PcdCfgDatabaseSize)
+};
+
+CONST DEBUG_LOG_BUFFER_HEADER mLogBufHdrTmpl = {
+  DEBUG_LOG_BUFFER_SIGNATURE,
+  sizeof (DEBUG_LOG_BUFFER_HEADER),
+  0,
+  {0, 0},
+  sizeof (DEBUG_LOG_BUFFER_HEADER),
+  FixedPcdGet32 (PcdEarlyLogBufferSize)
+};
+
+
+/**
+  Get flash map info.
+
+  @param[in]   BufInfo    BUF_INFO structure pointer to return flash map info
+
+  @retval      The flash map pointer.
+               NULL if flash map is invalid.
+**/
+STATIC
+FLASH_MAP *
+GetFlashMapBufInfo (
+  IN   BUF_INFO     *BufInfo

```

Comment:
Looks BufInfo is output, so use OUT instead of IN.

Reply:
OK, will change it.